sys/dev/pci/pci_user.c

```diff
@@ -121,7 +123,14 @@ struct pci_conf_io64 {
 	uint32_t		status;		/* request status */
 };
 
+struct pci_list_vpd_io64 {
+	struct pcisel	plvi_sel;	/* device to operate on */
+	uint64_t	plvi_len;	/* size of the data area */
+	uint64_t	plvi_data;
+};
+
 #define	PCIOCGETCONF64	_IOC_NEWTYPE(PCIOCGETCONF, struct pci_conf_io64)
+#define	PCIOCLISTVPD64	_IOC_NEWTYPE(PCIOCLISTVPD, struct pci_list_vpd_io64)
 /*
  * We don't support PCIOCGETCONF_OLD64 because the only COMPAT_FREEBSD64
  * architectures didn't exist until 9.0 (mips).
```

Comment:
This comment is a bit stale now...

Reply:
Only the list of architectures?  We still don't implement that particular ioctl I believe.

Reply:
Yeah. It could be changed to "the _earliest_ COMPAT_FREEBSD64 architecture".

Reply:
Done as an extra commit in the series.